docker/demo_server/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       - files:/usr/src/conforma-server/build/files
       - localisation:/usr/src/conforma-server/build/localisation
       - preferences:/usr/src/conforma-server/build/preferences
+      - ${BACKUPS_FOLDER:-backups}:/usr/src/conforma-server/build/backups
```

Comment:
No sure why adding an suffix `-backups` to the BACKUP_FOLDER. Better to leave without a suffix IMO

Reply:
That's not a suffix -- it's a fallback if `BACKUPS_FOLDER` is not provided ;)